config/locales/en.yml

```diff
@@ -217,6 +217,7 @@ en:
         signed_up_but_unconfirmed: "A message with a confirmation link has been sent to your email address. Please open the link to activate your account."
         unknown_error: "Something went wrong while creating your account. Check your email address and try again."
     failure:
+      disabled: "Your account has been disabled. Please contact an administrator to solve this issue."
```

Comment:
I decided to put this translation on `devise` side even if not totally related. Tell me if there is better option.